main.go

```diff
@@ -39,7 +38,7 @@ func ndkDownloadURL(revision string) string {
 	return fmt.Sprintf("https://dl.google.com/android/repository/android-ndk-r%s-%s-x86_64.zip", revision, runtime.GOOS)
 }
 
-func installedNDKVersion(ndkHome string) string {
+func ndkRevision(ndkHome string) string {
```

Comment:
Nit: `ndkHome` parameter should be `ndkPath`, because it's not necessarily an "NDK home directory".